buyer-api/src/operations/createDataOrder.js

```diff
@@ -26,7 +27,10 @@ export async function createDataOrder(dataOrder) {
   // TODO: use env.BUYER_PUBLIC_BASE_URL instead
   const buyerUrl = `${dataOrder.buyerUrl}/orders/${id}/offchain-data`;
   const headsUpUrl = `${dataOrder.buyerUrl}/orders/${id}/heads-up`;
-  const dataResponsesUrl = `${dataOrder.buyerUrl}/orders/${id}/data-responses`;
+  let dataResponsesUrl = `${dataOrder.buyerUrl}/orders/${id}/data-responses`;
+  if (config.env === 'development') {
+    dataResponsesUrl = `http://10.0.2.2:9100/orders/${id}/data-responses`;
```

Comment:
There should be a better way to do this.

Reply:
Why do we need to check this anyway? Since you have to create orders for each environment, just send `http://10.0.2.2:9100` as `dataOrder.buyerUrl`, instead of hard-coding it here.

Reply:
You are right guys, we discussed this one with @CarrilloJuan and we are going to improve this.